virtcontainers/qemu.go

```diff
@@ -534,6 +534,7 @@ func (q *qemu) createSandbox(ctx context.Context, id string, networkNS NetworkNa
 
 	rtc := govmmQemu.RTC{
 		Base:     "utc",
+		Clock:    "host",
```

Comment:
please document this, why `host`? and use the constant defined in govmm `RTCClock`

Reply:
host is the default value if clock is not set. This code just try to fix the logic error, but keep the behavior not changed.